src/services/createRouter.spec.ts

```diff
@@ -82,7 +82,7 @@ test('route update updates the current route', async () => {
 
 })
 
-test('route is readonly except for individual params', async () => {
+test.fails('route is readonly except for individual params', async () => {
```

Comment:
are you sure we want to assert this fails? I would probably just opt for removing the test

Reply:
My thought is I probably want to bring back this functionality in a future PR. So I'd like to keep it for now.